travis-ci/vmtest/build_pahole.sh

```diff
@@ -15,7 +15,7 @@ cd ${REPO_PATH}
 git init
 git remote add origin ${PAHOLE_ORIGIN}
 git fetch origin
-git checkout master
+git checkout v1.19
```

Comment:
Is it necessary to have a fixed version here? At least for the floating-point types on s390 it would be great to still use the master branch.

Reply:
that was temporary workaround for pahole build failing due to using some new DWARF constant

I sort of abandoned this PR, but can try to resurrect it next week and see if I still run into the same set of problems